fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1219,6 +1217,10 @@ public Builder setRunner(@Nullable FDBDatabaseRunner runner) {
             return this;
         }
 
+        private void setRunnerDefaults() {
+            setPriority(FDBTransactionPriority.BATCH);
+        }
```

Comment:
I also considered setting a default `WeakReadSemantics` here. In particular, I think you could make it one that set causal read risky but didn’t use cached read versions even if they are tracked, and I think that would be safe. The online indexer benefits from being able to see its own commits, but I think if it misses someone else’s, that’s generally fine (as all things are committed, so they are indistinguishable from just the timing being a little different).